tests/test_shell.py

```diff
@@ -113,9 +130,13 @@ def test_statepoint(self):
         self.call('python -m signac init my_project'.split())
         self.call(['python', '-m', 'signac', 'job', '--create', '{"a": 0}'])
         project = signac.Project()
-        for job in project:
-            sp = self.call('python -m signac statepoint {}'.format(job).split())
-            assert project.open_job(json.loads(sp)) == job
+        job = project.open_job({'a': 0})
+        sp = self.call('python -m signac statepoint {}'.format(job).split())
```

Comment:
Let's make it explicit that we use the job id here.
```suggestion
        sp = self.call('python -m signac statepoint {}'.format(job.id).split())
```